SPECS/libepoxy/libepoxy.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "libepoxy-1.5.5.tar.xz": "261663db21bcc1cc232b07ea683252ee6992982276536924271535875f5b0556"
+  "libepoxy-1.5.10.tar.xz": "072cda4b59dd098bba8c2363a6247299db1fa89411dc221c8b81b8ee8192e623"
```

Comment:
It seems the new archive is a `*tar.gz`, so we'd need to update this line as well.

Reply:
Moved to a different PR